### DISCUSSION TITLE
DevToolsActivePort file doesn't exist

### DISCUSSION THREAD

**Discussion by nirmalgoswami** [php-webdriver/php-webdriver#964] [Q&A] [answered]
i have started driver using `chromedriver` command

it is stated successfully but when i run code i got following error

```
Facebook\WebDriver\Exception\UnknownErrorException
unknown error: Chrome failed to start: crashed. (unknown error: DevToolsActivePort file doesn't exist) (The process started from chrome location /snap/chromium/1878/usr/lib/chromium-browser/chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
```
my code

```
  $options = (new ChromeOptions);
  $options->addArguments(['--no-sandbox ','--disable-dev-shm-usage']);
  $capabilities = DesiredCapabilities::chrome()->setCapability(ChromeOptions::CAPABILITY, $options);
  $driver = retry(5, function () use($capabilities) {
      return RemoteWebDriver::create('http://localhost:9515', $capabilities);
  }, 50);
  $browser = new Browser($driver);
  $browser->visit('https://www.google.com');
  $browser->quit();
```

google chrome version is 97.0.4692.99
google driver version is 97.0.4692.99
platform - ubuntu

**Comment by RestOp** [comment]
The same. Have you solved it?

**Comment by nirmalgoswami** [comment]
my bad, ubuntu does not support UI in web server so start without UI i.e. background process

P.S : In windows, you will able to see browser but in ubuntu, we can not browser as it is command line OS in server

**Comment by RestOp** [comment]
i am staring it with "--headless" attribute and get this error...

**Comment by humpataa** [comment, marked as answer]
had the same problem, I have added option "--no-sandbox" to get it working
